test/tests/p5.Gain.js

```diff
@@ -0,0 +1,51 @@
+'use strict '
+
+define(['chai'],(chai)=>{
+    const expect = chai.expect ;
+    describe('P5.Gain',()=>{
+        it('can be initilized and disposed',()=>{
+            let gain = new p5.Gain();
+            gain.dispose();
+        });
+        it('can set input', ()=>{
+            let inputNode = new p5.Gain();
+            let mainGainNode  = new p5.Gain();
+
+            mainGainNode.setInput(inputNode);
+        });
+        it('can send output',()=>{
+            let outputNode = new p5.Gain();
+            let mainGainNode  = new p5.Gain();
+            
+            mainGainNode.connect(outputNode);
+        });
+        it('can disconnect  all output' ,()=>{
+            let inputNode = new p5.Gain();
+            let outputNode = new p5.Gain();
+            let mainGainNode  = new p5.Gain();
+
+            mainGainNode.setInput(inputNode);
+            mainGainNode.connect(outputNode);
+            mainGainNode.disconnect();
```

Comment:
this is testing that we are able to call the `disconnect` method without that throwing any runtime errors, but how do we know if we successfully disconnected? (I'm not sure!)

Also, will this disconnect the inputNode or only the connection between the mainGaneNode and the outputNode? I think it's only that one. Again, how would we know from this test?